### ISSUE TITLE
1.1.1:  test_wcs.py fails with wcslib-5.13

### ISSUE THREAD

**Issue by olebole** [astropy/astropy#4535]
Hi,

with the new wcslib version 5.13, `test_validate` fails with
```
________________________________ test_validate _________________________________
>   ???
E   assert set(['', "'le...se SIP', ...]) == set(['', "'len...se SIP', ...])
E     Extra items in the left set:
E     "- 'unitfix' made the change 'Changed units: 'HZ      ' -> 'Hz''."
E     Extra items in the right set:
E     "- 'unitfix' made the change 'Changed units: 'HZ' -> 'Hz''."
E     Use -v to get the full diff

/usr/lib/python2.7/dist-packages/astropy/wcs/tests/test_wcs.py:435: AssertionError
```
This happens for both Python 2 and 3. Full test log is [here](https://ci.debian.net/data/packages/unstable/amd64/p/python-astropy/20160127_061935.autopkgtest.log.gz).

**Comment by toddrme2178** [comment]
I can confirm this.

**Comment by nden** [comment]
It looks like the number of spaces in the warning message has changed. 
I'll package 5.13 with a fix later today.

**Comment by olebole** [comment]
@nden I am using wcslib as an external dependency (Debian packaging), therefore I'd ask not to fix it in wcslib, but the test.

**Comment by nden** [comment]
@olebole That's right. I am going to fix the test. And package 5.13 with astropy.

**Comment by nden** [comment]
Although the fix for the test should go also in 1.0.9 so may be separate PRs is better.

**Comment by olebole** [comment]
As there is not wcslib-5.14 out with a fix of the flex memory leak, it would be worth to use that instead.

**Comment by astrofrog** [comment]
@nden - just to check, will you open a PR to update WCSLIB to 5.14? (once we do upgrade we can close https://github.com/astropy/astropy/issues/4441)

**Comment by nden** [comment]
@astrofrog See #4579.

**Comment by nden** [comment]
Closed by #4564
